### DISCUSSION TITLE
Does it modify my database?

### DISCUSSION THREAD

**Discussion by abitrolly** [mindsdb/mindsdb#2279] [General]
https://docs.mindsdb.com/ doesn't explicitly mention if my data is safe when I connect to my server with MindsDB. It is okay to read the data, but I am worried that CREATE statement may alter the database config or add tables that will break things at some point.

**Comment by StpMax** [comment]
Hi @abitrolly 
Current version of MindsDB use user databases only for reading data. We plan to add the "insert into..." statement in the future. At the moment, only way to create table in user database is use MindsDB indirectly through [dbt-mindsdb](https://pypi.org/project/dbt-mindsdb/).

**Reply by abitrolly** [reply]
Thanks. Probably worths adding a link to the intro explaining that.
